api/apps/geoprocessing/src/migrations/geoprocessing/1708524274000-AddStableIdsToFeaturesDataRows.ts

```diff
@@ -0,0 +1,44 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddStableIdsToFeaturesDataRows1708524274000
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+DROP TRIGGER tr_precompute_feature_property_list ON features_data;
+
+CREATE TRIGGER tr_precompute_feature_property_list AFTER INSERT ON features_data
+FOR EACH ROW EXECUTE
+PROCEDURE precompute_feature_property_list();
```

Comment:
You' basically removing the AFTER UPDATE condition here, right? What's the reason for it? (mostly because I'm not 100% sure on what this trigger does, and I would like to know the difference)

Reply:
that's correct - switching from a setup where we trigger `AFTER INSERT OR UPDATE` to only triggering `AFTER INSERT`. In practice, there's no need to ever execute `precompute_feature_property_list()` more than once and `ON INSERT`, as spatial data in `features_data` can never actually change, by design (we don't have a way to _update_ `features_data` rows).

On the other hand, keeping the `AFTER UPDATE` part of the trigger setup would pointlessly recalculate feature property lists when the migration would do the simple update to set the stable id for existing data - this would take forever even on a local development instance with not much spatial data in it, so I took the chance to simplify this a bit, which also makes this whole migration run snappily rather than taking ages and doing a lot of repeated work.